lib/Minion/Backend/Pg.pm

```diff
@@ -222,22 +222,28 @@ sub retry_job {
 }
 
 sub stats {
-  my $self = shift;
+  my ($self, $queue) = (shift, shift // '');
```

Comment:
An options hash would be more future proof here.

Reply:
I was thinking about it and couldn't imagine any scenario where stats are needed for something different than single queue. So because of that and for simplicity of usage I'd stick to plain single optional parameter.